mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -212,11 +213,12 @@ impl mayastor_server::Mayastor for MayastorSvc {
                     match lvol.share_nvmf(None).await {
                         Ok(s) => {
                             debug!("created and shared {} as {}", lvol, s);
+                            sync_config().await;
```

Comment:
This is not needed, so really no need to sync the config here. Also please consider in what context we are running this in

Reply:
removed